test/metabase/api/search_test.clj

```diff
@@ -285,15 +286,26 @@
       (is (= [] (:available_models (mt/user-http-request :crowberto :get 200 "search?q=noresults")))))))
 
 (deftest query-model-set-test
-  (with-search-items-in-root-collection "query-model-set"
-    (testing "should returns a list of models that search result will return"
-     (is (= #{"dashboard" "dataset" "segment" "collection" "action" "metric" "card"}
-            (set (mt/user-http-request :crowberto :get 200 "search/models" :q "query-model-set")))))
-    (testing "return a subsets of model for created-by filter"
-      (is (= #{"dashboard" "dataset" "card" "action"}
-             (set (mt/user-http-request :crowberto :get 200 "search/models"
-                                        :q "query-model-set"
-                                        :created_by (mt/user->id :rasta))))))))
+  (let [search-term "query-model-set"]
+    (with-search-items-in-root-collection search-term
+      (testing "should returns a list of models that search result will return"
+        (is (= #{"dashboard" "dataset" "segment" "collection" "action" "metric" "card"}
+               (set (mt/user-http-request :crowberto :get 200 "search/models" :q search-term)))))
+      (testing "return a subsets of model for created-by filter"
+        (is (= #{"dashboard" "dataset" "card" "action"}
+               (set (mt/user-http-request :crowberto :get 200 "search/models"
+                                          :q search-term
+                                          :created_by (mt/user->id :rasta))))))
+      (testing "return a subsets of model for verified filter"
```

Comment:
What about a test for when the premium feature flags aren't present? For example, without the official-collections feature flag, but with the content-verification flag, the result should be `#{"card"}`

Reply:
added those here https://github.com/metabase/metabase/blob/caa17f36206a1cc74a5e7d52022bb2e9fb5ef28e/test/metabase/api/search_test.clj#L310